docs/how_to_make_a_tahoe-lafs_release.org

```diff
@@ -1,80 +1,97 @@
-    necessary (TODO: pushing just 1.12.0b2 was insufficient: travis fired,
-    but not buildbot)
-    - git push official master TAGNAME
-    - that will build tarballs
-  - [ ] make sure buildbot is green
-  - [ ] download tarballs+wheels
-  - [ ] announce tree is unlocked
-  - [ ] compare tarballs+wheels against local copies (but sign upstreams)
-  - [ ] sign each with "gpg -ba -u 68666a7a FILE"
-  - [ ] twine upload dist/*
-  - [ ] test "pip install tahoe-lafs" (from PyPI)
-  - [ ] upload *.asc to org ~source/downloads/
-  - [ ] copy the release tarball,sigs to tahoe-lafs.org: ~source/downloads/
-  - [ ] move old release out of ~source/downloads (to downloads/old/?)
-  - [ ] send out relnotes.txt:
-    - add prefix with SHA256 of tarballs, release pubkey, git revhash
-    - GPG-sign the email with release key
-    - send to tahoe-announce@tahoe-lafs.org and tahoe-dev@tahoe-lafs.org
-  - [ ] update Wiki front page: version on download link, News column
-  - [ ] update Wiki "Doc": parade of release notes (with rev of NEWS.rst)
-  - [ ] close the Milestone on the trac Roadmap
-  - [ ] tweet to @tahoelafs
-  - other stuff:
-  - [ ] update https://tahoe-lafs.org/hacktahoelafs/
-  - [ ] make an "announcement of new release" on freshmeat
+  - [ ] added final release name and date to top-most item in NEWS.rst
+  - [ ] updated relnotes.txt
+  - [ ] updated CREDITS
+  - [ ] updated docs/known_issues.rst
+  - [ ] docs/INSTALL.rst only points to current tahoe-lafs-X.Y.Z.tar.gz source code file
+  - [ ] updated https://tahoe-lafs.org/hacktahoelafs/
+
+* sign + build the tag [0/8]
+
+  - [ ] code passes all checks / tests (i.e. all CI is green)
+  - [ ] documentation is ready (see above)
+  - [ ] git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-X.Y.Z" tahoe-lafs-X.Y.Z
+  - [ ] build code locally:
+     tox -e py27,codechecks,coverage,deprecations,docs,integration,upcoming-deprecations
+  - [ ] created tarballs (they'll be in dist/ for later comparison)
+     tox -e tarballs
+  - [ ] release version is reporting itself as intended version
+     ls dist/
+  - [ ] 'git pull' doesn't pull anything
+  - [ ] pushed tag to trigger buildslaves
+     git push official master TAGNAME
+  - [ ] confirmed Dockerhub built successfully:
+     https://hub.docker.com/r/tahoelafs/base/builds/
+
+* sign the release artifacts [0/8]
+
+  - [ ] pushed signed tag (should trigger Buildbot builders)
+  - [ ] Buildbot workers built all artifacts successfully
+  - [ ] downloaded upstream tarballs+wheels
+  - [ ] announce on IRC that master is unlocked
+  - [ ] compared upstream tarballs+wheels against local copies
+  - [ ] signed each upstream artifacts with "gpg -ba -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A FILE"
+  - [ ] added to relnotes.txt: [0/3]
+    - [ ] prefix with SHA256 of tarballs
+    - [ ] release pubkey
+    - [ ] git revision hash
+  - [ ] GPG-signed the release email with release key (write to relnotes.txt.asc)
```

Comment:
.. and another signing task.  This one specifies no key.  Is it also for Brian or can someone else at least sign an email telling people there are some signed artifacts they can go find?